databricks/koalas/base.py

```diff
@@ -839,7 +839,7 @@ def astype(self, dtype):
             raise ValueError("Type {} not understood".format(dtype))
         return self._with_new_scol(self.spark.column.cast(spark_type))
 
-    def isin(self, values):
+    def isin(self, values) -> "ks.Series":
```

Comment:
`Union["ks.Series", "ks.Index"]`?

Reply:
Sounds good! Updated the document as well!

Reply:
Nice!